client/plots/violin.renderer.js

```diff
@@ -478,8 +478,7 @@ function getLegendGrps(self) {
 					hiddenOpacity: 1
 				})
 			}
```

Comment:
See my comment above on not sharing variable across functions and loops, unless necessary.